app/assets/javascripts/fortitude/jquery/blocks/shade.js

```diff
@@ -1,36 +1,22 @@
 (function($) {
-    'use strict';
-    $(document).
-      on('open:ft:shade', '.shade', function(event) {
-        var $this = $(this);
-        if(!$this.hasClass('shade--is-active')) {
-          $this
-            .addClass('shade--is-active')
-            .addClass($this.data().showClass)
-            .show()
-            .waitForAnimation()
-            .then(function() {
-              $this.removeClass($this.data().showClass);
-            });
-        }
-      }).
-      on('close:ft:shade', '.shade', function(event) {
-        var $this = $(this);
-        if($this.hasClass('shade--is-active')) {
-          $this
-            .addClass($this.data().hideClass)
-            .waitForAnimation()
-            .then(function() {
-              $this
-                .hide()
-                .removeClass('shade--is-active')
-                .removeClass($this.data().hideClass);
```

Comment:
we need these back its what allows the consumer to tell which class is used for animation eg `fadeOut`